task_test.go

```diff
@@ -755,6 +755,41 @@ func TestIncludesCallingRoot(t *testing.T) {
 	tt.Run(t)
 }
 
+func TestIncludesOptional(t *testing.T) {
```

Comment:
What does this do? I looked at https://github.com/go-task/task/blob/8f80fc4e2c283c41cb1525b88996bd07743cbf31/task_test.go#L24 but it's not clear to me.

Reply:
@juampynr it's a helper to test whether the contents of a file match an expected output. In the case of the one immediately below, it's checking that `testdata/includes_optional/called_dep.txt` exists and contains the text `called_dep`. This was generated by `testdata/includes_optional/Taskfile.yml` running `echo "called_dep" > called_dep.txt` as a command